### ISSUE TITLE
Send Line to Terminal

### ISSUE THREAD

**Issue by fightthepower** [geany/geany#2174]
I use geany for python programming with vte enabled. I regularly use `Send Selection to Terminal` for my work and I find it most helpul for debugging my code. `Send Selection to Terminal`  works every time except the  Indented lines (eg:`<tab>print('world')`) where it result in an `IndentationError`.   I also request a function in which we can send a line of code to terminal without even selecting it. For example 

```
i=0
print("hello")
if i==0:
    print("world")
```
if  I want to print hello, now I need to select the line `print("hello")` and then `Send Selection to Terminal` to get the output in vte. Instead of this I want to click any where on that line and then do `Send Line to Terminal` without selecting it. Similarly to print world instead of selecting `<tab>print('world')` I want to click on that line and then print the output world.

**Comment by lcorag** [comment]
I also agree on this. I used to use Geany in exactly the same way. I think that the current "Send Selection to Terminal" action would work fine if setting `terminal.send_selection_unsafe` to `true` would work something link adding a newline at the end of the selection, rather than after each line of code
